### PR TITLE
feat: Removing fetch-node-details sdk dependency from example app.

### PR DESCRIPTION
**Pull request by grvgoel81**
Removing fetch-node-details sdk dependency from example app.

### CODE REVIEW

**Review by metalurgical — APPROVED** [review]

**Review by chaitanyapotti — APPROVED** [review]
lgtm

File: singlefactorauth/src/main/java/com/web3auth/singlefactorauth/SingleFactorAuth.kt [hunk 1]
```diff
@@ -42,9 +45,25 @@ class SingleFactorAuth(
...
+                    sfaCF.complete(
+                        gson.fromJson(
+                            JSONObject(response).toString(),
+                            SFAKey::class.java
+                        )
+                    )
+                }
+            }
+        }
+        return sfaCF
```
Comment (metalurgical) [inline singlefactorauth/src/main/java/com/web3auth/singlefactorauth/SingleFactorAuth.kt:66]: Why did you do this?

File: singlefactorauth/build.gradle [hunk 2]
```diff
@@ -40,11 +40,11 @@ dependencies {
 
     implementation 'org.web3j:crypto:4.8.8-android'
     implementation 'org.torusresearch:torus-utils-java:4.0.2'
-    implementation 'org.torusresearch:fetch-node-details-java:5.0.0'
+    api 'org.torusresearch:fetch-node-details-java:5.0.0'
```
Comment (metalurgical) [inline singlefactorauth/build.gradle:43]: Only this change was supposed to go in.

File: singlefactorauth/src/main/java/com/web3auth/singlefactorauth/SingleFactorAuth.kt (hunk 1 above)
Comment (grvgoel81) [reply singlefactorauth/src/main/java/com/web3auth/singlefactorauth/SingleFactorAuth.kt:66]: Done some changes in session-manager-sdk and this sdk. We can discuss those. This is done to return the correct error from initialize function.

Comment (metalurgical) [reply singlefactorauth/src/main/java/com/web3auth/singlefactorauth/SingleFactorAuth.kt:66]: We had already discussed the problem with doing it this way and that it is not a substitute for proper error handling..

**Pull request merged**